components/input/ColorPicker.js

```diff
@@ -19,21 +19,22 @@ const ColorPicker = ({ text, initialRgbaColor, onColorChange, ...rest }) => {
     };
     const handleChange = (color) => {
         setRgbaColor(color.rgb);
+        if (onChange) {
```

Comment:
You can use a noop as default if you want via proptypes. (noop is inside proton-shared/funcitons)

Reply:
Sounds good! I included `noop` in the new commit.